.env

```
@@ -2,4 +2,6 @@ JWT_SECRET=iBtpvMhlzMyeN4LbX2TfycJTUQFD17SUoWlnPwtQ3uU
 JWT_ACCESS_EXPIRATION=3600000
 JWT_REFRESH_EXPIRATION=604800000
 JWT_ISSUER=skillscan-ai
-JWT_AUDIENCE=skillscan-web
\ No newline at end of file
+JWT_AUDIENCE=skillscan-web
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_AUDIENCE key should go before the JWT_ISSUER key

(UnorderedKey)